HeterogeneousCore/CUDAUtilities/src/CachingHostAllocator.h

```diff
@@ -393,18 +388,18 @@ namespace notcub {
         }
 
         // Done searching: unlock
-        mutex.Unlock();
+        mutex.unlock();
       }
 
       // Allocate the block if necessary
       if (!found) {
         // Attempt to allocate
         // TODO: eventually support allocation flags
-        if (CubDebug(error = cudaHostAlloc(&search_key.d_ptr, search_key.bytes, cudaHostAllocDefault)) ==
+        if ((error = cudaHostAlloc(&search_key.d_ptr, search_key.bytes, cudaHostAllocDefault)) ==
```

Comment:
This is a functional change, earlier it would print out the error code if `CUB_STDERR` was defined (which IIRC we do define). Doesn't really matter to me though.